<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q_Textsearching</class>
 <widget class="QWidget" name="Q_Textsearching">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>千词斩 Version 1.0</string>
  </property>
  <widget class="QPlainTextEdit" name="line_Text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>270</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>峰峰的锐眼</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>100</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输入文本：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_cet4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string>cet-4</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_cet6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cet-6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_toefl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>toefl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Search">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Back">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
